providers/implementations/signature/dsa_sig.c

```diff
@@ -951,9 +951,6 @@ static int dsa_sigalg_set_ctx_params(void *vpdsactx, const OSSL_PARAM params[])
     if ((ret = dsa_common_set_ctx_params(pdsactx, params)) <= 0)
         return ret;
 
-    if (params == NULL)
```

Comment:
Not sure why this check has been removed - if `params` comes into this function as `NULL`, `dsa_common_set_ctx_params()` will return `1`, which is not <= 0, so we won't have returned on line 952

Reply:
good point, will move the check upwards